package.json

```diff
@@ -26,11 +26,11 @@
   },
   "homepage": "https://github.com/actions/first-interaction#readme",
   "dependencies": {
-    "@actions/core": "file:toolkit/actions-core-0.0.0.tgz",
-    "@actions/exec": "file:toolkit/actions-exec-0.0.0.tgz",
-    "@actions/github": "file:toolkit/actions-github-0.0.0.tgz",
-    "@actions/io": "file:toolkit/actions-io-0.0.0.tgz",
-    "@actions/tool-cache": "file:toolkit/actions-tool-cache-0.0.0.tgz"
+    "@actions/core": "^1.10.0",
```

Comment:
Something tells me they (whoever from github reviews this) are going to want to just update those binary packages where necessary. It's clear from the code that they want to keep this standalone (ie, not requiring downloading external packages)

Reply:
Why do you think they would want that? Those local packages still seem to pull in other external dependencies.

Reply:
They clearly designed this to be less dependant on external packages. Doing that makes this application significantly less likely to fail when npm goes down or networking gets wonked in an action.

Reply:
I also think it is better not to set a specific version of those binary packages.